gcp-data-drive/bq.go

```diff
@@ -37,7 +37,7 @@ type bqDataPlatform struct {
 	query *bigquery.Query
 }
 
-// getData contains the implementation detail for retriving and marshaling data from Bigqueyr into JSON.
+// getData contains the implementation detail for retriving and marshaling data from Bigquery into JSON.
```

Comment:
Whoops, `BigQuery` (the Q is capitalized).

Reply:
I correct all occurrences in this file.